beta/docker/README.md

```diff
@@ -0,0 +1,231 @@
+
+   ```sh
+   docker swarm init # --advertise-addr 192.0.2.1
+   ```
+
+   > **Note**
+   >
+   > If Docker returns an error about multiple network interfaces, uncommment the `--advertise-addr` parameter and replace the example IP (`192.0.2.1`) with the public IP address of the Linux server.
+
+5. Clone this repository and switch to this directory:
+
+    ```sh
+    git clone https://github.com/1Password/scim-examples.git
+    cd ./scim-examples/beta/docker
+    ```
+
+6. Save your `scimsession` file to the working directory.
+
+   > **Note**
+   >
+   > If you are deploying SCIM bridge directly on the Linux server, you will need to transfer the `scimsession` file
+   > from your local machine that is signed in to your 1Password account, for example:
+   >
+   > ```sh
+   > scp ./scimsession user@host:scim-examples/beta/docker/scimsession
+   > ```
+
+### Configure 1Password SCIM bridge to connect to Google Workspace
+
+If integrating 1Password with Google Workspace, additional cofiguration is required. See [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client) for instructions to create the service account, key, and API client.
+
+1. Copy the Google Workspace service account key to the working directory.
+
+   > **Note**
+   >
+   > If you are deploying SCIM bridge directly on the Linux server, you will need to transfer the Workspace credentials file
+   > from your local machine that is signed in to Google (or your 1Password account if you saved the credentials there), for example:
+   >
+   > ```sh
+   >  scp ./workspace-credentials.json user@host:scim-examples/beta/docker/workspace-credentials.json
+   > ```
+
+2. Copy the [settings file template](/beta/workspace-settings.json) from this repository:
+
+    ```sh
+    cp ../workspace-settings.json ./workspace-settings.json
+    ```
+
+    Edit the file. Replace the values for each key:
+
+    - `actor`: the email address for the administrator that the service account is acting on behalf of
+    - `bridgeAddress`: the URL for your SCIM bridge based on the DNS record created in [Get started](#get-started)
+
+    ```json
+    {
+        "actor":"admin@example.com",
+        "bridgeAddress":"https://scim.example.com"
+    }
+    ```
+
+    Save the file.
+
+## Deploy 1Password SCIM bridge
+
+After the DNS record has propagated, set the domain name for your SCIM bridge inline, use the template to output a canonical configuration that will create a Docker secret and deploy SCIM bridge, then create the stack from this configuration. You SCIM bridge should automatically acquire and manage a TLS certificate from Let's Encrypt on your behlaf using the supplied domain name.
+
+*Example command:*
+
+```sh
+OP_TLS_DOMAIN=scim.example.com docker stack config --compose-file ./compose.template.yaml |
+    docker stack deploy --compose-file - op-scim-bridge
+```
```

Comment:
We're already pulling in `scim.env` as an env-file. We should be having users edit those rather than potentially setting them up for a poor situation where `OP_TLS_DOMAIN` is set multiple places.

Reply:
There's a method to my madness here based on [order of precedence for environment variables in Docker](https://docs.docker.com/compose/environment-variables/envvars-precedence/), but it could potentially use some tweaking for sure.

I have intentionally been trying to avoid editing files at all, especially for the most common use case. Setting variables inline means you can deploy in one step without editing anything. Providing `scim.env` with default config allows customers to use it as a base for their own customized setup if they want. It's also useful as a reference to hold information about the things that may be configured, either by editing this file or setting values inline.